glossary.html

```diff
@@ -24,46 +24,58 @@
 </head>
 
 <body>
-    <div class="initial-scale=1" value=width-wrap>
+    <div class="width-wrap">
         <header>
             <nav>
                 <ul>
                     <li>
-                        <p><a href="index.html">Home</a></p>
+                        <a href="index.html">Home</a>
                     </li>
                     <li>
-                        <p><a href="links.html">Links</a></p>
+                        <a href="links.html">Links</a>
                     </li>
                     <li>
-                        <p><a href="glossary.html">Glossary</a></p>
+                        <a href="glossary.html">Glossary</a>
                     </li>
                 </ul>
             </nav>
         </header>
         <main>
             <h1>Glossary</h1>
-            <dl>CLASS - HTML elements can have one or more classes, separated by spaces. You can style elements using
-                CSS by selecting them with their classes.</dl>
-            <dl>ID - An HTML element can have an id attribute to identify it. id elements should always be unique to
-                that single element, and each element should never have more than one id.</dl>
-            <dl>HREF - Links tell the browser where to go using an href attribute, which stores a URL.</dl>
-            <dl>Basic Formatting - You can easily format text to be bold, italic, or underlined using simple formatting
-                tags.</dl>
-            <dl>Body - The body is the container for all of a page’s content. Comes after the
+            <dl>
+                <dt>CLASS</dt>
+                <dd>HTML elements can have one or more classes, separated by spaces. You can style elements using
+                    CSS by selecting them with their classes.</dd>
```

Comment:
Definition lists consist of one `<dl>` element that contains pairs of `<dt>` and `<dd>` elements, as corrected here